src/quo2/theme.cljs

```diff
@@ -22,3 +25,31 @@
   ([light-value dark-value override-theme]
    (let [theme (or override-theme (get-theme))]
      (if (= theme :light) light-value dark-value))))
+
+(defn provider
+  "Wrap `children` in a React Provider using `quo2.theme/theme-context` as the
+  context.
+
+  `options`: Clojure map. Currently we only use the `:theme` key. In the future
+  we may support other settings.
+  "
+  [options & children]
+  (into [:> (.-Provider theme-context) {:value (clj->js options)}]
```

Comment:
@J-Son89 The call to `clj->js` is unnecessary. I tested without it now, and `options` will always be stored as a JS object in the React Context.